docs/CactbotCustomization.md

```diff
@@ -294,7 +294,43 @@ and also reading through existing triggers in [ui/raidboss/data](../ui/raidboss/
 
 ## Overriding Raidboss Timelines
 
-Sorry, this is hard to do right now.
+Overriding a raidboss timeline is similar to [overriding a trigger](#overriding-raidboss-triggers).
+
+The steps to override a timeline are:
+
+1) Copy the timeline text file out of cactbot and into your user folder
+
+  For example, you could copy [ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt](../ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt).
+
+1) Add a section to your user/raidboss.js file to override this.
```

Comment:
```suggestion
2) Add a section to your user/raidboss.js file to override this.
```

Reply:
Actually these are pretty funny (read: not funny at all); so, Markdown in general lets you do the following for numbered lists:

```md
1) Test
1) Test
```
And that will be:
1) Test
1) Test

But, because the numbered items have paragraphs between them that are improperly indented (Markdown says 3 or more is the minimum for indentation), these do actually come out to be:

```md
1.
1.
1.
```
In the preview.

So, either:
```md
1) Point

  Text

2) Point

  Text
```
or:
```md
1) Point

   Text

1) Point

   Text
```
Thanks Markdown 🙄

Reply:
How very interesting. No, that auto-numbering was something I wasn't aware of. At least it's a "simple" fix, right?

Reply:
I wouldn't necessarily say that writing your own markup language to replace Markdown entirely is simple, but... 😏